### PR TITLE
Support RGBDS 0.8.0 object version 10

### PR DESCRIPTION
**Pull request by Rangi42**
I'm sure those `Some`s can be simplified.

Once this is approved, we can update rgbobj and have a new release in sync with RGBDS.

### CODE REVIEW

**Comment by Rangi42** [comment]
(This is also making me wonder, while we can still revise pre-release RGBDS, if the version number actually needs incrementing from 9 to A? or if the revision from 9 to 10 would be fine by itself)

**Comment by ISSOtm** [comment]
Yeah, I think just a revision increment is sufficient.

@aaaaaa123456789 has some ideas for a new, more forward-compatible object file format. We'll use that for 1.0.0, but that's later.

File: src/lib.rs [hunk 1]
```diff
@@ -11,7 +11,10 @@
 //!
 //! RGBDS release                                          | Object file format
 //! -------------------------------------------------------|-------------------
-//! [v0.6.0](https://rgbds.gbdev.io/docs/v0.?.?/rgbds.5)   | v9 r9
+//! [v0.8.0](https://rgbds.gbdev.io/docs/v0.8.0/rgbds.5)   | vA r10
+//! [v0.7.0](https://rgbds.gbdev.io/docs/v0.7.0/rgbds.5)   | v9 r9
```
Comment (ISSOtm) [inline src/lib.rs:15]:
```suggestion
//! [v0.7.0](https://rgbds.gbdev.io/docs/v0.7.0/rgbds.5)   | Reports v9 r9, but really is v9 r10.
```

**Comment by Rangi42** [comment]
> Yeah, I think just a revision increment is sufficient.

Okay, then https://github.com/gbdev/rgbds/pull/1413 will need merging first.

**Comment by ISSOtm** [comment]
Done!

**Comment by Rangi42** [comment]
And done!

**Review by ISSOtm — APPROVED** [review]
Thank you! This will get published together with the next RGBDS release (as the object file format *could* change again til then, at least in theory).

**Pull request merged**